Angular/angular.md

```diff
@@ -3,30 +3,26 @@
 
 ## Motivation
 
-Angular is a popular Javascript Framework that is being used to build Applications for Web, Desktop and Mobile Devices. Launched by Google in 2010 as AngularJS, the subsequent versions of AngularJS are simply called Angular and involves the usage of Typescript which is a superset of Javascript. Typescript is compatible with Javascript and compiles to Javascript which is rendered on the Browser.
+- Angular is a popular Javascript Framework that is being used to build Applications for Web, Desktop and Mobile Devices. Launched by Google in 2010 as AngularJS, the subsequent versions of AngularJS are simply called Angular and involves the usage of Typescript which is a superset of Javascript. Typescript is compatible with Javascript and compiles to Javascript which is rendered on the Browser.
```

Comment:
I meant to say that convert this paragraph into small bullet points, not the entire paragraph to single bullet point.